test/testTrainingFunction.php

```diff
@@ -0,0 +1,162 @@
+<?php
+require '../trainingFunction.php';
+use PHPUnit\Framework\TestCase;
+class testTrainingFunction extends TestCase {
+
+    public function testGetAllDatesSuccess_1(){
+        $db_array = [
+                [
+                    'date'=>'2022-07-10',
+                    'exercise'=>'Max Hang',
+                    'difficulty'=>'hard',
+                    'weight_added_kg'=>50,
+                    'comments'=>'blah blah blah'
+                ],
+                [
+                    'date'=>'2022-04-06',
+                    'exercise'=>'Repeater',
+                    'difficulty'=>'easy',
+                    'weight_added_kg'=>10,
+                    'comments'=>'these are some words'
+                ]
+            ];
```

Comment:
it would be good to add another item here that has the same date as one of these ones, that way you are also testing that the functions removes duplicates